.eslintrc.yml

```diff
@@ -0,0 +1,60 @@
+extends:
+- airbnb-base
+- next
+
+env:
+  browser: true
+  es2021: true
+  es6: true
+  jest: true
+
+globals:
+  google: readonly
+  module: readonly
+  process: readonly
+
+parserOptions:
+  ecmaVersion: 8
+  sourceType: module
+  ecmaFeatures:
+    jsx: true
+
+rules:
+  arrow-body-style: off
+  arrow-parens: off
+  func-names: warn
+  func-style: off
+  max-len: warn
+  no-bitwise: off
+  no-cond-assign:
+  - warn
+  - except-parens
```

Comment:
I added [this parameter](https://eslint.org/docs/rules/no-cond-assign) because it disables intrinsic JavaScript functionality without which some useful patterns are impossible, and the only justification it gives is that people might accidentally use `=` instead of `===`...